ocrd-tool.json

```diff
@@ -18,7 +18,20 @@
       "tags": ["Text recognition and optimization"],
       "description": "Recognize text in lines with tesseract",
```

Comment:
Is it lines only?

Reply:
At the moment it is. Walk the regions, walk the lines within the regions, recognize text in the bounding box of the line. To make that configurable was the idea of the textequiv_level parameter.